src/rockstor/storageadmin/views/rockon_id.py

```diff
@@ -145,13 +145,11 @@ def post(self, request, rid, command):
                         cco.val = cc_map[c]
                         cco.save()
                     for e in env_map.keys():
-                        if (not DContainerEnv.objects.filter(
+                        if (DContainerEnv.objects.filter(
```

Comment:
> my pr is not hiding anything

While writing my last answer i thought about possible "misunderstandings" and I really hope you did not see it like a judgement :)
My suggestion: I think you can master this issue and your last post confirms this idea :tada: - with "master this issue" I mean you can think about db redesign, code refactoring, etc etc...and I really hope you'll do it!!! We really appreciate new contributors :wink: